docs/configuration-options.md

```diff
@@ -0,0 +1,124 @@
+# Mend Renovate Configuration Options
+
+## Configure Mend Renovate
+
+Mend Renovate Enterprise Edition runs with one or more **_Server_** containers and one or more **_Worker_** containers.
+Mend Renovate Community Edition runs on a single Server container that also performs the Worker actions. 
+See below for a list of environment variables that relate to each.
+
+### Environment variables for Community Edition and Enterprise Server
+
+The following environment variables apply to **Mend Renovate Community Edition** and the **Mend Renovate Enterprise Edition Server**.<br/>
+Environment variables for the **Mend Renovate Enterprise Worker** are in the next section.
+
+#### Mend licensing config
+
+**`MEND_RNV_ACCEPT_TOS`**: Set this environment variable to `y` to consent to [Mend's Terms of Service](https://www.mend.io/terms-of-service/).
+
+**`MEND_RNV_LICENSE_KEY`**: Contact Mend to request a license key. [Mend.io/renovate](https://www.mend.io/renovate/)
```

Comment:
This needs a more specific link to the Community Edition form. I couldn't not find a link to it from mend.io/renovate (which should probably be fixed)

Reply:
Updated to https://www.mend.io/renovate-community/